examples/device/cdc_msc/src/tusb_config.h

```diff
@@ -102,8 +102,8 @@
 #define CFG_TUD_VENDOR           0
 
 // CDC FIFO size of TX and RX
-#define CFG_TUD_CDC_RX_BUFSIZE   64
-#define CFG_TUD_CDC_TX_BUFSIZE   64
+#define CFG_TUD_CDC_RX_BUFSIZE   (TUD_OPT_HIGH_SPEED ? 512 : 64)
```

Comment:
There is actually more discussion on this #423 , I plan to complete the fix after this stm hs branch is merged.

Reply:
Should TUD_OPT_HIGH_SPEED  be also TUD_OPT_RHPORT. I have left as it for now.

Reply:
No, `TUD_OPT_HIGH_SPEED` is the correct choice, TUD_OPT_RHPORT = 1 doesn't means it is high speed. It is possible that the OTG_HS port operates with FS using internal PHY ( HS mode requires external PHY chip). Also this is platform-independent configuration, STM32 specific port selection cannot dictate the config file.